docs/src/app/app-routes.jsx

```diff
@@ -92,39 +92,54 @@ const AppRoutes = (
       <Route path="auto-complete" component={AutoCompletePage} />
       <Route path="avatar" component={AvatarPage} />
       <Route path="badge" component={BadgePage} />
+      <Redirect from="buttons" to="/components/buttons/flat-button" />
+      <Route path="buttons">
```

Comment:
It looks like it's for the `initiallyOpen`. I would rather fix this at the List components level.